renku/api/repository.py

```diff
@@ -368,3 +368,11 @@ def track_paths_in_storage(self, *paths):
                 stderr=STDOUT,
                 cwd=str(self.path),
             )
+
+    def pull_path(self, path):
```

Comment:
can you support multiple paths with one `run`?

Reply:
I would need to check that the argument is either a glob or an existing path -- any `click` methods that can do that for me? :) Or do you mean just taking a list of paths?

Reply:
just taking list of paths:

```
* -I <paths> --include=<paths>:
  Specify lfs.fetchinclude just for this invocation; see [INCLUSION & EXCLUSION]
```

Reply:
I did that here: https://github.com/SwissDataScienceCenter/renku-python/pull/333/files#diff-0a809905e68e7794f7dc630519b636f5R40 

But I can move the iteration over the paths to this method instead.